internal/entity/common.go

```diff
@@ -227,3 +227,18 @@ type Metadata struct {
 	UpdatedBy int64     `json:"updated_by"`
 	DeletedAt time.Time `json:"deleted_at,omitempty"`
 }
+
+type StateFilterType int
+
+const (
+	Active  StateFilterType = 0
```

Comment:
@reviewers - wouldn't it be better to use string value instead of int for that? Please help me with your opinion.